packages/auth/src/Expo-AuthSession/README.md

```diff
@@ -0,0 +1,114 @@
+# EXPO AUTH SESSION
+
+An authentication library which uses Expo Auth Session under the hood, and requires you to set up
+and app registration in Azure. This replaces the previous implementation that is using
+`react-native-msal`.
+
+# Usage
+
+If you are already using `mad-auth 0.1.*` via the `mad-core` package here are some guidelines for migrating to `mad-auth 1.0`:
+
+You have to install `expo-auth-session` and `expo-crypto` in your project.
+
+```
+npm i expo-auth-session expo-crypto
+```
+
+```
+yarn add expo-auth-session expo-crypto
+```
+
+Next, add this to your `mad.config.ts` file:
+
+```ts
+    experimental: {
+        useExpoAuthSession: true
+    }
+```
+
+the `mad-core` package should handle most of the authentication logic for you. If there are certain usages in your app that use any mad-auth functions directly, 
+you can import these functions directly by adding 
+``import { ExpoAuthSession } from "@equinor/mad-auth;"``
+and prepending ``ExpoAuthSession.`` to the methods you currently use from ``mad-auth
+
```

Comment:
I would move `mad-core` related documentation to `mad-core`'s readme. For a random consumer this information is not relevant

Reply:
fixed